code/modules/clothing/glasses/glasses.dm

```diff
@@ -155,4 +155,29 @@
 	name = "Optical Thermal Eyepatch"
 	desc = "An eyepatch with built-in thermal optics."
 	icon_state = "eyepatch"
-	item_state = "eyepatch"
\ No newline at end of file
+	item_state = "eyepatch"
+
+/obj/item/clothing/glasses/cold
+	name = "cold goggles"

```

Comment:
Are you fucking kidding me danno, the description outright says "meant for low temperatures"

Remove that shit or make it actually help with cold.